### PR TITLE
[chore](BE) add more log for better tracing for be write

### PR DESCRIPTION
**Pull request by ByteYue**
# Proposed changes

Issue Number: close #xxx
Recently when tracing when bug happened in version1.1.4 I found out there were some places we can add more log for a better tracing.
## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/exec/tablet_sink.cpp [hunk 1]
```diff
@@ -503,6 +516,8 @@ void NodeChannel::try_send_batch(RuntimeState* state) {
 
     // tablet_ids has already set when add row
     request.set_packet_seq(_next_packet_seq);
+    LOG(INFO) << "try to send " << row_batch->num_rows() << " row to " << this->_node_id
```
Comment (morningman) [inline be/src/exec/tablet_sink.cpp:519]: That will print too much logs

File: be/src/olap/memtable_flush_executor.cpp [hunk 2]
```diff
@@ -42,7 +42,10 @@ std::ostream& operator<<(std::ostream& os, const FlushStatistic& stat) {
 OLAPStatus FlushToken::submit(const std::shared_ptr<MemTable>& memtable) {
     RETURN_NOT_OK(_flush_status.load());
     int64_t submit_task_time = MonotonicNanos();
-    _flush_token->submit_func(std::bind(&FlushToken::_flush_memtable, this, memtable, submit_task_time));
+    auto st = _flush_token->submit_func(std::bind(&FlushToken::_flush_memtable, this, memtable, submit_task_time));
+    if (UNLIKELY(!st.ok())) {
+        return OLAP_ERR_OTHER_ERROR;
```
Comment (morningman) [inline be/src/olap/memtable_flush_executor.cpp:47]: Better print st.get_error_msg() here.

**Review by morningman — CHANGES_REQUESTED** [review]
This PR will print too much logs.
If you want to add log to debug temporarily, I suggest to commit in your own repo and test your own branch.

**Comment by ByteYue** [comment]
> This PR will print too much logs. If you want to add log to debug temporarily, I suggest to commit in your own repo and test your own branch.

Sorry for so many annoying logs. I do agree it's too many logs and some of them might be redundant. And I do want to seek some help from reviews about which log should be added😂. 
I intend to add some useful logs like 1. printing the rows processed by add_batch 2. _pending_batches_bytes when closing the node_channel 3. skipped tablet id when add batch 4. print the afterwards log when it's the last rpc 5. print the logic that has something to do with eos.
The intention to add these logs is that recently we are encountered with one strange bug that one txn's logs show there were no delta writers for a node channel on one client BE. But it's curious cause when you open one node channel there must be at least one tablet and corresponding delta writer. And the afterwards load channel destructor log even show there is no memory usage indicating there is no write. It's so strange.

**Comment by morningman** [comment]
i don't know which log can help you to debug your issue. but for me, this should be debug log.

my point is，how about adding these logs to a temp branch and run it in your env to debug？

File: be/src/exec/tablet_sink.cpp (hunk 1 above)
Comment (ByteYue) [reply be/src/exec/tablet_sink.cpp]: Already delete this line along with a lot of useless log lines

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**